.github/workflows/main.yml

```diff
@@ -30,7 +30,7 @@ env:
   TOOLS_DIR: 'C:\tools'
 
   # PHP extensions required by Composer
-  EXTENSIONS: apcu, gettext, gd, igbinary, imagick, intl, json, mbstring, msgpack, memcached, sqlite3, yaml, redis
+  EXTENSIONS: apcu, gettext, gd, igbinary, imagick, intl, json, mbstring, msgpack, memcached, sqlite3, yaml, redis, :memcache
```

Comment:
Is it correct syntax with `:`?

Reply:
Yes. That ensures that we remove `memcache` so that it does not interfere with `memcached` and the codeception tests